### ISSUE TITLE
Getting rate limited with FAST_MODE=0

### ISSUE THREAD

**Issue by xologram** [mrrfv/cloudflare-gateway-pihole-scripts#126]
Trying to run cf_list_create.js and keep getting rate limited by CF around chunk 68 

```
Created "CGPS List - Chunk 68" list - 119 left
An error occured while making a web request: "SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON", retrying. Attempt 1 of 50.
```

I tried lowering CLOUDFLARE_LIST_ITEM_LIMIT but same thing happens. What am I doing wrong here?

**Comment by mrrfv** [comment]
You're doing nothing wrong, this is an ongoing problem caused by a sudden change to the Cloudflare API. Can you confirm that you are using the latest CGPS version?

**Comment by xologram** [comment]
yep i just cloned the repo and used it for the first time. i ended up giving  [#111 (comment)](https://github.com/mrrfv/cloudflare-gateway-pihole-scripts/issues/111#issuecomment-2151733701 a shot and it worked.

**Comment by rodeodomino** [comment]
I'm still getting this issue too. Thanks!

**Comment by mrrfv** [comment]
This is expected for now. The script should continue normally if you give it time to retry enough times.

**Comment by shukryshuk** [comment]
may iknow how long will it take for 200chunks if im using #111  ???
